demisto_sdk/commands/download/downloader.py

```diff
@@ -700,17 +710,20 @@ def log_files_downloaded(self) -> None:
         """
         log_msg, added_msg, merged_msg = str(), str(), str()
         if self.num_added_files:
-            added_msg = f'{self.num_added_files} files added'
+            files = 'file' if self.num_added_files == 1 else 'files'
+            added_msg = f'{self.num_added_files} {files} added'
         if self.num_merged_files:
-            merged_msg = f'{self.num_merged_files} files merged'
+            files = 'file' if self.num_merged_files == 1 else 'files'
+            merged_msg = f'{self.num_merged_files} {files} merged'
         if added_msg:
             if merged_msg:
                 log_msg = f'\n{added_msg}, {merged_msg}.'
             else:
                 log_msg = f'\n{added_msg}.'
-        else:
+        elif merged_msg:
             log_msg = f'\n{merged_msg}.'
-        print_color(log_msg, LOG_COLORS.NATIVE)
+        if log_msg:
+            print_color(log_msg, LOG_COLORS.NATIVE)
```

Comment:
If you have both `merged_msg` and `log_msg` you will print only the `merged_msg `. why?